index.html

```diff
@@ -75,8 +75,10 @@ <h2>Heading</h2>
           <p><a class="btn btn-default" href="#" role="button">View details &raquo;</a></p>
        </div>
         <div class="col-md-4">
-          <h2>Heading</h2>
-          <p>Donec sed odio dui. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Vestibulum id ligula porta felis euismod semper. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus.</p>
+          <h2>Content Presentation Guidelines</h2>
+          <p><li>1) Registered Users have the ability to upload both images as well as a file that has text,</li>
```

Comment:
Rather than list bullet items here within the paragraph element, try  changing the paragraph elements to an ordered list and remove the numbers from each line.
```suggestion
          <ol>
          <li>Registered Users have the ability to upload both images as well as a file that has text,</li>
```